src/terrain/terrain.js

```diff
@@ -1422,7 +1419,7 @@ export class Terrain extends Elevation {
     /*
      * Bookkeeping if something gets rendered to the tile.
      */
-    prepareDrawTile(coord: OverscaledTileID) {
+    prepareDrawTile(coord: OverscaledTileID) { // eslint-disable-line no-unused-vars
```

Comment:
Would changing that to:
```
prepareDrawTile(_: OverscaledTileID) {
```
achieve the same?

Reply:
I also realize that we may be able to just remove it entirely and replace for `prepareDrawTile()`.

Reply:
Removing the argument makes Flow unhappy, and using _ is less descriptive I feel. Ideally, those methods would be refactored to not require those arguments, but I left that for a follow up since I'm not very familiar with the code.

Reply:
Gave it a quick look, feel free to cherry-pick dc9729918018093feae8a25814d2a746d77c905d.

Reply:
Thanks! If we do this, is prepareDrawTile still a good name for what the method does?

Reply:
I think so, although a bit generic `prepareDrawTile` simply notifies to the renderer that the currently processed tile (for deferred/draping mode) is ready and about to be rendered.